app/controllers/support/devices_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Support::DevicesController < Support::BaseController
```

Comment:
I found the name DevicesController confusing, as this doesn't create Devices - isn't this more like a `Support::Devices::KeyContactController` ? 
I think we're going to be using the Support::Devices namespace for a few more things soon

Reply:
Yes good call, wasn't sure what to call it while it was the only controller in the namespace